src/commands/index.ts

```diff
@@ -59,48 +59,68 @@ function toggleAngularPresetCommand() {
   const preset = 'angular';
   const value = getToggleValue(preset);
   const message = `${msg.ngPresetMessage} ${value ? msg.enabled : msg.disabled}. ${msg.restart}`;
-  togglePreset(preset, value, false);
-  showCustomizationMessage(message, [{ title: msg.reload }], applyCustomization, cancel, preset, !value, false);
+  const values = getConfig().inspect(`vsicons.presets.${preset}`);
+  const defaultValue = values.defaultValue as boolean;
+  const initValue = values.workspaceValue as boolean;
+  togglePreset(preset, value, defaultValue, false);
+  showCustomizationMessage(message, [{ title: msg.reload }],
+    applyCustomization, cancel, preset, !value, initValue, false);
 }
 
 function toggleJsPresetCommand() {
   const preset = 'jsOfficial';
   const value = getToggleValue(preset);
   const message = `${msg.jsOfficialPresetMessage} ${value ? msg.enabled : msg.disabled}. ${msg.restart}`;
-  togglePreset(preset, value);
-  showCustomizationMessage(message, [{ title: msg.reload }], applyCustomization, cancel, preset, !value);
+  const values = getConfig().inspect(`vsicons.presets.${preset}`);
```

Comment:
I thought the returned types were `any`. The truth is that I haven't tested it with the real code. Just thought of it while reviewing it here.